lib/measured/base.rb

```diff
@@ -27,6 +36,10 @@ def method_missing(method, *args)
 require "measured/arithmetic"
 require "measured/unit"
 require "measured/conversion"
+require "measured/conversion_builder"
 require "measured/conversion_table"
 require "measured/measurable"
-require "measured/case_sensitive_measurable"
+  
+module Measured
+  private_constant :Measurable
```

Comment:
I really don't like this. For a few reasons. In ruby you can't actually hide anything, as it's always possible to get to anyway. Sometimes easily and by accident:

```ruby
pry(main)> Measured::Measurable
NameError: private constant Measured::Measurable referenced
from (pry):2:in '__pry__'
pry(main)> Measured::Length
=> Measured::Length
pry(main)> Measured::Length.superclass
=> Measured::Measurable
```